server/services/store/sqlstore/blocks.go

```diff
@@ -14,32 +15,23 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 )
 
-func (s *SQLStore) latestsBlocksSubquery(c store.Container) sq.SelectBuilder {
-	internalQuery := sq.Select("*", "ROW_NUMBER() OVER (PARTITION BY id ORDER BY insert_at DESC) AS rn").From(s.tablePrefix + "blocks")
-
-	return sq.Select("*").
-		FromSelect(internalQuery, "a").
-		Where(sq.Eq{"rn": 1}).
-		Where(sq.Eq{"delete_at": 0}).
-		Where(sq.Eq{"coalesce(workspace_id, '0')": c.WorkspaceID})
-}
-
 func (s *SQLStore) GetBlocksWithParentAndType(c store.Container, parentID string, blockType string) ([]model.Block, error) {
 	query := s.getQueryBuilder().
 		Select(
 			"id",
 			"parent_id",
 			"root_id",
 			"modified_by",
-			"schema",
+			"`schema`",
 			"type",
 			"title",
-			"COALESCE(\"fields\", '{}')",
+			"COALESCE(`fields`, '{}')",
```

Comment:
Do you know for sure that backticks also work on SQLite and Postgres, or did you test on those? I just don't know if they do or not, and got burnt by SQL quoting issues on previous commits. If you know this works, that's good with me.

Reply:
Hmmm... a quick search seems to indicate [backticks are non-standard](https://wiki.postgresql.org/wiki/Things_to_find_out_about_when_moving_from_MySQL_to_PostgreSQL)? We should test on all 3 DBs to be sure.

Reply:
Yes, clearly the backtick is a problem specially for postgres. I'll fix that, anyway yes, we need to test all the databases before it.

Reply:
Now is fixed :)